tests/models/test_dagrun.py

```diff
@@ -135,11 +135,11 @@ def create_dag_run(
 
         return dag_run
 
-    def test_clear_task_instances_for_backfill_running_dagrun(self, session):
+    def test_clear_task_instances_for_backfill_running_dagrun(self, dag_maker, session):
         now = timezone.utcnow()
         state = DagRunState.RUNNING
         dag_id = "test_clear_task_instances_for_backfill_running_dagrun"
-        with DAG(dag_id=dag_id) as dag:
+        with dag_maker(dag_id=dag_id) as dag:
             EmptyOperator(task_id="backfill_task_0")
         self.create_dag_run(dag, logical_date=now, is_backfill=True, state=state, session=session)
```

Comment:
Can probably use the dagmaker create dagrun tooling too.

Reply:
Could result in a lot of tests rewrite